### PR TITLE
Small changes to improve PureKerasModel serialization

### PR DESCRIPTION
**Pull request by mcgibbon**
Two functions in the precipitative model are turned into layers, so that later I can look at the configuration of a save-loaded model and compare it to the configuration of starting model to ensure PureKerasModel is serializing correctly.

A get_config method is added to a normalization layer which takes an initialization argument, so it can be used within PureKerasModel. Without this method, keras complains it requires this method to serialize the layer.

Resolves #<github issues> [JIRA-TAG]

(Delete this for the commit message)
You are encouraged to check the PR against the [Code Review Checklist](https://paper.dropbox.com/doc/Code-Review-Checklist--A4lKrs~xg7w5Gsb39N6JLNQoAg-IlsYffZgTwyKEylty7NhY).

### CODE REVIEW

File: external/fv3fit/fv3fit/keras/_models/precipitative.py [hunk 1]
```diff
@@ -35,27 +35,21 @@
...
-    return tf.math.scalar_mul(
-        # dQ2 * delp = s-1 * Pa = s^-1 * kg m-1 s-2 = kg m-1 s-3
-        # dQ2 * delp / g = kg m-1 s-3 / (m s-2) = kg/m^2/s
-        tf.constant(-1.0 / GRAVITY, dtype=dQ2.dtype),
-        tf.math.reduce_sum(tf.math.multiply(dQ2, delp), axis=-1),
-    )
-
+class IntegratePrecipLayer(tf.keras.layers.Layer):
+    def call(self, args) -> tf.Tensor:
+        dQ2, delp = args  # layers seem to take in a single argument
```
Comment (frodre) [inline external/fv3fit/fv3fit/keras/_models/precipitative.py:40]: What was the error when using multiple args?  I've used layers before with specific arguments for each tensor and I think it worked.

https://github.com/ai2cm/fv3net/blob/d4924b40618a363a5ad27a98b6fb410f541d7a48/external/fv3fit/fv3fit/emulation/layers/fields.py#L89

File: external/fv3fit/fv3fit/keras/_models/precipitative.py [hunk 2]
```diff
@@ -35,27 +35,21 @@
...
-    """
-    Args:
-        dQ2: rate of change in moisture in kg/kg/s, negative corresponds
-            to condensation
 
-    Returns:
-        heating rate in degK/s
-    """
-    return tf.math.scalar_mul(tf.constant(-LV / CPD, dtype=dQ2.dtype), dQ2)
+class CondensationalHeatingLayer(tf.keras.layers.Layer):
```
Comment (frodre) [inline external/fv3fit/fv3fit/keras/_models/precipitative.py:50]: I would keep the documentation describing the specifics of the values used/output by this layer.

File: external/fv3fit/fv3fit/keras/_models/precipitative.py (hunk 1 above)
Comment (frodre) [reply external/fv3fit/fv3fit/keras/_models/precipitative.py:40]: Descriptions are embedded in comments here.  Make consistent with `CondensationalHeatingLayer`

**Review by frodre — APPROVED** [review]
LGTM.  Minor comments to check out the multiple argument input and make the layer documentation consistent.

File: external/fv3fit/fv3fit/keras/_models/precipitative.py (hunk 1 above)
Comment (mcgibbon) [reply external/fv3fit/fv3fit/keras/_models/precipitative.py:40]: There wasn't an error, I just (back when I wrote the original code here, the comment is pre-existing) noticed keras seems to have a "style" where most layers take in a list of inputs, instead of separate arguments.

Thanks for catching the dropped docstrings, added docstrings to both layers.

File: external/fv3fit/fv3fit/keras/_models/precipitative.py (hunk 2 above)
Comment (mcgibbon) [reply external/fv3fit/fv3fit/keras/_models/precipitative.py:50]: Restored, thanks.
